### PR TITLE
Create html based report for dry runs for cloud providers.

### PR DESCRIPTION
**Pull request by msajidmansoori12**
This PR adds functionality to create html based reports for dry runs for AWS, AZURE, GCP. 

Sample report attached. 
Screenshot 2024-07-10 at 16-16-11 Cloud resources page (1)

### CODE REVIEW

**Review by jyejare — COMMENTED** [review]
I would love to see Resource Types being on `Rows` and the data of them in the next column.

File: cloudwash/utils.py [hunk 1]
```diff
@@ -24,47 +39,64 @@ def echo_dry(dry_data=None) -> None:
...
+        "skipped_vms": dry_data["VMS"]["skip"],
+        "deletable_discs": dry_data["DISCS"]["delete"],
+        "deletable_nics": dry_data["NICS"]["delete"],
+        "deletable_images": dry_data["IMAGES"]["delete"],
+        "deletable_pips": dry_data["PIPS"]["delete"] if "PIPS" in dry_data else None,
+        "deletable_resources": dry_data["RESOURCES"]["delete"],
+        "deletable_stacks": dry_data["STACKS"]["delete"] if "STACKS" in dry_data else None,
+    }
+    if any(value for key, value in resource_data.items() if key != 'provider'):
+        logger.info("Resources eligible for cleanup:")
```
[the remark this answers is not in the document]
Comment (msajidmansoori12) [reply cloudwash/utils.py]: Removed.

File: cloudwash/utils.py [hunk 2]
```diff
@@ -24,47 +39,64 @@ def echo_dry(dry_data=None) -> None:
...
+        for key, value in resource_data.items():
+            if value and key != "provider":
+                logger.info(
```
[the remark this answers is not in the document]
Comment (msajidmansoori12) [reply cloudwash/utils.py]: Makes sense, refactored the code to print the same way.

**Comment by msajidmansoori12** [comment]
Screenshot 2024-10-03 at 12-52-58 Cloud resources page
After incorporating the comments , the output turns out to be like the table above.

**Pull request merged**